kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/JdbcConnectionParams.java

```diff
@@ -115,6 +115,7 @@ public class JdbcConnectionParams {
   // Non-configurable params:
   // Currently supports JKS keystore format
   static final String SSL_TRUST_STORE_TYPE = "JKS";
+  static final String SSL_STORE_PASSWORD_PATH = "storePasswordPath";
```

Comment:
I think it should be classified to the previous param group? just at the next line of SSL_KEY_STORE_PASSWORD

Reply:
add an additional blank line before this variable definition